spaceflights-pandas/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/data_processing/nodes.py

```diff
@@ -17,7 +17,7 @@ def _parse_money(x: pd.Series) -> pd.Series:
     return x
 
 
-def preprocess_companies(companies: pd.DataFrame) -> pd.DataFrame:
+def preprocess_companies(companies: pd.DataFrame) -> tuple[pd.DataFrame, dict]:
```

Comment:
How come the return type changed when the returned argument is still the same?

Reply:
good point, it might have had happened when i move the files from the viz starters folder to here. It's prob what defined in the viz starters file